crates/ln-dlc-node/src/node/mod.rs

```diff
@@ -1070,6 +1078,8 @@ async fn update_rgs_snapshot(
 
 impl Display for NodeInfo {
     fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
-        format!("{}@{}", self.pubkey, self.address).fmt(f)
+        let scheme = if self.is_ws { "wss" } else { "tcp" };
+
+        format!("{scheme}://{}@{}", self.pubkey, self.address).fmt(f)
```

Comment:
♥️

Reply:
I've realized this is actually technically wrong and should be changed to just `ws` :sweat_smile:

Reply:
Fixed